### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by hemslo**
Add [k6](https://k6.io/) tests, tested 2 scenarios:

* can access homepage without token
* can get features with token

More tests will be added later once the way of token management is clear for upgrade.

### CODE REVIEW

File: integration_test/test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+export function homepage () {
+  const res = http.get(UNLEASH_BASE_URL)
+
+  check(res, {
+    'homepage response code was 200': (res) => res.status === 200
+  })
+}
+
+export function apiClientFeatures () {
```
Comment (janboll) [inline integration_test/test.js:36]: What if you remove the token, does it still give 200?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+KC_CLIENT_SECRET="ZYhToGucoRMwaWHxXyGKsjunXlXV6Uto"
```
Comment (janboll) [inline .env.example:1]: this is just an example, right? :laughing:

Comment (hemslo) [reply .env.example:1]: yes, it's moved from `docker-compose` file, already included in https://github.com/app-sre/unleash/blob/master/.gitleaks.toml

File: integration_test/test.js (hunk 1 above)
Comment (hemslo) [reply integration_test/test.js:36]: 401, added one more test case in https://github.com/app-sre/unleash/pull/54/commits/729ca278f00c04840951f27a5842f04f0f6a04c3

**Review by janboll — APPROVED** [review]

**Pull request merged**